servers/rendering/renderer_rd/shaders/gi.glsl

```diff
@@ -355,65 +355,70 @@ void sdfgi_process(vec3 vertex, vec3 normal, vec3 reflection, float roughness, o
 			vec3 ray_dir = reflection;
 
 			{
-				float prev_radius = cascade > 0 ? radius_sizes[cascade - 1] : 0.0;
+				float prev_radius = cascade > 0 ? radius_sizes[max(0, cascade - 1)] : 0.0;
```

Comment:
There's already a check for `cascade > 0` so the `max()` seems unnecessary.

Reply:
Ah, I forgot that test in.